terraform/aws/modules/storage/main.tf

```diff
@@ -20,7 +20,7 @@ resource "aws_security_group" "km_rds_sg" {
     from_port   = 5432
     to_port     = 5432
     protocol    = "tcp"
-    cidr_blocks = ["0.0.0.0/0"]
+    cidr_blocks = ["<cidr>"]
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure Security Groups do not have unrestricted specific ports open - Postgres SQL (TCP,5432)**
 
**Vulnerability**
Security Groups - Unrestricted Specific Ports - Postgres SQL (TCP,5432). Exposed ports may allow attackers to access the application listening on that port.
 
**Remediation**
In AWS Console - 
 1. Sign in the AWS Console and go to the Amazon VPC console.
 2. In the navigation pane, select Security Groups.
 3. For each security group, perform the following actions:
      a) Select the security group.
      b) Select the Inbound Rules.
      c) Identify the rules to be removed.
      d) Remove the required rules.
 4. Click Save.

source: accurics-app